packages/website/components/layout.js

```diff
@@ -31,7 +31,7 @@ export default function Layout({
   const shouldWaitForUser = needsUser && status === 'loading'
 
   return (
-    <div className="sans-serif">
+    <div className="sans-serif flex flex-column min-vh-100">
```

Comment:
Now that we have this can you remove `min-vh-100` from the other pages?

Reply:
Yes, good call 👍

Reply:
Actually, if we remove the min-vh-100 from the secondary container in all the pages, they don't grow to fill the remaining space...

Reply:
You're right, it needs additional work. We can handle that later 👍